cypress/support/fragments/marcAuthority/marcAuthority.js

```diff
@@ -122,6 +138,10 @@ export default {
     cy.expect(buttonLink.exists());
     cy.expect(Callout('Field 700 has been linked to a MARC authority record.').exists());
   },
+  checkLinkingAuthorityByTag: (tag) => {
```

Comment:
Create quickmarc editing methods in quickMarcEditor.js file, but before check there please there are methods already created for this type of actions.

Reply:
All same methods in file have hardcoded row index in message.

Reply:
fixed